resolwe_bio/tools/make_report.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python2
```

Comment:
We avoided using Python 3 in this case because the script imports matplotlib, which is not installed for for Python3 in the docker image. We plan to improve this in the future.